routing/turns_tts_text_i18n.cpp

```diff
@@ -0,0 +1,187 @@
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include <regex>
+#include <string>
+
+namespace routing
+{
+namespace turns
+{
+namespace sound
+{
+
+long FindInStrArray(std::u32string_view haystack, char32_t needle)
```

Comment:
1. The code that uses it doesn't need long, it needs bool.
2. Can the code below be replaced with haystack.find(needle) call?